src/cookbook/networking/background-parsing.md

```diff
@@ -38,9 +38,12 @@ First, add the [`http`][] package to your project.
 The `http` package makes it easier to perform network
 requests, such as fetching data from a JSON endpoint.
 
-```yaml
-dependencies:
-  http: <latest_version>
+To install the `http` package,
+add it to the dependencies section of the `pubspec.yaml` file
+by running `flutter pub add`:
```

Comment:
**suggestion:** Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
To install the `http` package, run `flutter pub add`.
This command adds `http` to the `dependencies` section of the `pubspec.yaml` file:
```